hack/nightly.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+NIGHTLY_TAG_FMT="+%m%d%y"
+RELEASE_REPO=${RELEASE_REPO:-public.ecr.aws/z3c6l6z9/karpenter-nightly}
+RELEASE_VERSION=${RELEASE_VERSION:-$(date "${NIGHTLY_TAG_FMT}")}
```

Comment:
Do we want to include what the most recent release version was here? Might be easier for tracking rather than having to cross reference release dates and nightly date tags.

Reply:
The tag needs to remain guessable, e.g. a simple date, adding more to it would make that impossible. It's easy to find what version was correlated since all version tags have a tag.

Reply:
I wonder if we can add multiple tags, we can revisit this when I add the Latest tags as part of the remaining things to do.